client/state/console-dispatch/index.js

```diff
@@ -15,15 +15,15 @@
 /**
  * Internal dependencies
  */
-import * as actionTypes from 'state/action-types';
+// import * as actionTypes from 'state/action-types';
```

Comment:
whats the change here for?

Reply:
the change is because when we inline action types we no longer have the ability to import all of them, though I guess this could still work. I'd just as soon eliminate this code though because I'm not sure how much it's used and even when it _is_ used I don't think it's all that valuable.